integration/internal/network/ops.go

```diff
@@ -20,10 +20,9 @@ func WithIPv4(enable bool) func(*network.CreateOptions) {
 }
 
 // WithIPv6 Enables IPv6 on the network
-func WithIPv6() func(*network.CreateOptions) {
+func WithIPv6(enable bool) func(*network.CreateOptions) {
```

Comment:
I'm slightly on the fence on adding the argument (but could use input from others); perhaps a `WithDisableIPv6()` could make it easier to find places where we disable, and we could keep the `WithIPv6` as-is.


Also curious what is the default if not set? (we don't have an option to reset, correct?)

Reply:
I added `WithDisableIPv6()` and removed the commit that was changing `WithIPv6()` signature.